### DISCUSSION TITLE
Community is where the user says it is

### DISCUSSION THREAD

**Discussion by TonyGravagno** [RealRaven2000/FiltaQuilla#285] [Ideas]
Now that we have this Discussions area in GitHub (Thank you!!!) I'll note that this will only be used by people who are inclined to create GitHub accounts. That means developers. As an "end-user" product, these addons are not just used by developers, they're used by people who have never written a line of code and don't know what a github is.

So in an effort to create actual community, where users are more likely to hang out, I recommend creating other resources for users, and linking to them from the addon pages.

Examples include:

- Discord : Preferred by a younger audience for quick responses when possible.
- Google Groups (GG) : More traditional long-form Q&A "forum".
- Facebook Groups (FG) : Definitely a place where average users are found, people who are not likely to have a Discord account or to subscribe to a GG.
- LinkedIn Groups : Like GG and FG, there is merit here, but it's mostly a business audience, and a lot of business tends to run on platforms like Microsoft Office. While it would be good to have a presence there, it should not be the only other presence, as it would still isolate average home users and others who are not inclined to have a business presence on LinkedIn.
- A Discourse subsite : Probably the best place to start. Completely neutral for all audiences. It's easy to setup as something like forum.quickfolders.org.
- Simple WordPress forum : This or any popular FOSS package.

I will volunteer to create, host, and manage any of these resources. Axel would have full ownership and admin rights so that no one else can run off with his resources.

Other suggestions might include Twitter or Slack but those resources have issues for a project like this : Twitter is uni-directional, broadcast-only, not good for exchanges. Slack has a limited free model now and messages disappear after a period of time.

Might Thunderbird.net already have a forum resource?  
Is there already someplace like ThunderbirdAddons.net where people discuss all addons? (If not, maybe there should be.)

The [Thunderbird (Support) Community](https://support.mozilla.org/en-US/products/thunderbird) might also be a good place for a recognized presence, but not as the one and only place. That's a place for problem reports and general help, not so much for addon-specific help. And the culture there isn't oriented toward discussion like this new Discussions area, or GG, or Discord.

There's always the concept of too many community resources fragmenting the ecosystem. But not enough, as we see now, means little or no ecosystem. I only recommend a better balance, and I'll be happy to help with this as I've done for many other software offerings like this.

**Comment by RealRaven2000** [comment]
interesting. I already use discord (admin by others) for music production and related activities, an Add-on users / enthusiast discord would probably be something I could also join if you were to set it up. It could be much broader and host discussions on all mail related Add-ons, maybe something I could mention on the next dev meeting to see whether the other add-on owners would also be interested. 

getting a community started is actually hard work it needs consistent input and moderation to thrive; so I won;t commit to specific outcomes right now. let this thought percolate for a while... (hey sorry getting tired here, it's 1am and typing on my wife's laptop is cumbersome)

**Reply by TonyGravagno** [reply]
I've done this many times before, am involved (own/manage/participate) in a number of communities, and am happy to do this. For me, this kind of effort isn't as time consuming or gut wrenching as for someone just getting started.

For our purposes I think a small Discord server dedicated to your addons is a prudent start. Expanding this with more grandiose goals (which is a tendency with which I struggle) to include other Tb addons would bring in a lot of new considerations. That would detract from the primary goal here of just getting people to communicate about your addons. Adding more people absolutely results in loss of some few who might just be there for one focused purpose. You wouldn't want one of your users to go away because some loudmouth is on a rant about Tb or some other addon.

So I'm thinking any groups should probably be just for your own addons. I / We / They can take on similar initiatives for a broader Tb addon audience later.

If you'd like to start with Discord, I'll fire it up and you can decide if you want to keep it.

**Reply by RealRaven2000** [reply]
> For our purposes I think a small Discord server dedicated to your addons is a prudent start. Expanding this with more grandiose goals (which is a tendency with which I struggle) to include other Tb addons would bring in a lot of new considerations. That would detract from the primary goal here of just getting people to communicate about your addons. Adding more people absolutely results in loss of some few who might just be there for one focused purpose. You wouldn't want one of your users to go away because some loudmouth is on a rant about Tb or some other addon.

one could have a room or thread for each of the Add-ons to keep the information focused. Let's put the thinking cap on when I am back from my holidays.
